mail_tracking/controllers/main.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import werkzeug
+from psycopg2 import OperationalError
+from openerp import api, http, registry, SUPERUSER_ID
+import logging
+_logger = logging.getLogger(__name__)
+
+BLANK = 'R0lGODlhAQABAIAAANvf7wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw=='
+
+
+class MailTrackingController(http.Controller):
+
+    def _request_metadata(self):
+        request = http.request.httprequest
+        return {
+            'ip': request.remote_addr,
+            'user_agent': request.user_agent,
+            'os_family': request.user_agent.platform,
+            'ua_family': request.user_agent.browser,
+        }
+
+    @http.route('/mail/tracking/open/<string:db>'
+                '/<int:tracking_email_id>/blank.gif',
+                type='http', auth='none')
+    def mail_tracking_open(self, db, tracking_email_id, **kw):
```

Comment:
Wouldn't it be better to overload [track_mail_open](https://github.com/odoo/odoo/blob/8.0/addons/mass_mailing/controllers/main.py#L10-L18)?